src/PhpStan/NamespaceConsistencyRule.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/*
+ * This file is part of the FiveLab CiRules package
+ *
+ * (c) FiveLab
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code
+ */
+
+declare(strict_types = 1);
+
+namespace FiveLab\Component\CiRules\PhpStan;
+
+use PhpParser\Node;
+use PHPStan\Analyser\Scope;
+use PHPStan\Rules\Rule;
+
+class NamespaceConsistencyRule implements Rule
```

Comment:
phpstan more checks logic, but not strings, correct names, etc... As result, for this we should add phpcs rule, not phpstan.